tests/scripts/run_performance.sh

```diff
@@ -65,7 +67,8 @@ run_perf_models_cnn_javelin() {
     local tt_arch=$1
     local test_marker=$2
 
-    echo "There are no CNN tests for Javelin yet specified. Arch $tt_arch requested"
+#    echo "There are no CNN tests for Javelin yet specified. Arch $tt_arch requested"
```

Comment:
Please delete this line - we need to stop the practice of commenting out things we don't need rather than deleting them.